javascript/eslint-config-node/README.md

```diff
@@ -0,0 +1,43 @@
+# @instea/eslint-plugin-node
+
+Default ESLint rules for Node.js projects.
+
+## Installation
+
+```
+yarn add --dev prettier eslint\
```

Comment:
Isn't it possible to bundle these plugins in the Instea package and so make the installation more straightforward?

Reply:
I have changed it to direct dependencies, should work just fine and it is simpler